edition/std

```diff
@@ -10,3 +10,5 @@ LIMA_INSTALL_LOGROTATE=true
 LIMA_INSTALL_OPENSSH_SFTP_SERVER=true
 LIMA_INSTALL_SSHFS=true
 LIMA_INSTALL_TINI=true
+
+LIMA_INSTALL_CGROUP_CONF=true
```

Comment:
Instead of 2 variables (`LIMA_INSTALL_CGROUP_CONF` and `LIMA_ALPINE_CGROUP_MODE`) I would prefer to have just a single variable `LIMA_CGROUP_MODE` that is set to the mode value, e.g.

```suggestion
LIMA_CGROUP_MODE=hybrid
```

But given that this is the default, this setting should not be made in `edition/std` at all. Instead there should be a line in `edition/min` that just documents that the variable exists:

```shell
export LIMA_CGROUP_MODE=
```

Note that I removed the `ALPINE_` substring from the variable name; it feels redundant to me, as every setting in this repo is about building an Alpine image.